lib/index.ts

```diff
@@ -265,14 +265,10 @@ export default class Backburner {
       }
     }
 
-    let onError = getOnError(this.options);
-
-    if (onError) {
-      try {
-        return method.apply(target, args);
-      } catch (error) {
```

Comment:
so `Ember.onerror` can't be used to inform bug tracking code for `Ember.run.bind` or `run.join`?  That doesn't seem right. I think the only think this needs to do is rethrow after the `onError` invocation to satisfy both constraints.

Reply:
Did you review the test cases I added? Which test is incorrect? Unless I am misunderstanding you, I have test cases showing run.join with and without an onerror (with and without that onerror re-throwing). Can you jot down a snippet that explains the scenario you are concerned with?

Reply:
> I think the only think this needs to do is rethrow after the onError invocation to satisfy both constraints.

FWIW, this would result in calling onerror twice with the same error. As I explained in the PR description, run.join’s callback is always already called from within a try catch that will properly defer to the provided onerror...

Reply:
oops you are correct!